routes/Routes.js

```diff
@@ -0,0 +1,159 @@
+import express, { response } from "express";
+import listEndpoints from "express-list-endpoints";
+import HappyThought from "../model/Happythoughts";
+const router = express.Router();
+
+// get 20 happy thoughts
+router.get("/thoughts", async (req, res) => {
+  try {
+    const happyThoughts = await HappyThought.find()
+      .sort({ createdAt: "desc" })
+      .limit(20)
+      .exec();
+    res.json(happyThoughts);
+  } catch (error) {
+    res.status(400).json({
+      success: false,
+      response: error,
+      message: "Could not get thoughts",
+    });
+  }
+});
+
+//post new happy thought
+router.post("/thoughts", async (req, res) => {
+  const { message } = req.body;
+  try {
+    const newHappyThought = await new HappyThought({ message }).save();
+    res.status(201).json(newHappyThought);
+  } catch (error) {
+    console.error(error); // Log the error
+    res.status(400).json({
+      success: false,
+      response: error,
+      message: "Could not save thought",
+    });
+  }
+});
+
+//post a like to a happy thought
+router.post("/thoughts/:thoughtId/like", async (req, res) => {
+  const { thoughtId } = req.params;
+  try {
+    const updatedThought = await HappyThought.findByIdAndUpdate(
+      thoughtId,
+      { $inc: { hearts: 1 } },
+      { new: true }
+    );
+    if (updatedThought) {
+      res.json(updatedThought);
+    } else {
+      res.status(404).json({
+        success: false,
+        response: error,
+        message: "Thought not found",
+      });
+    }
+  } catch (error) {
+    res.status(400).json({
+      success: false,
+      response: error,
+      message: "Could not save like",
+    });
+  }
+});
+
+//delete a happy thought
+router.delete("/thoughts/:thoughtId", async (req, res) => {
```

Comment:
Nice! Would be cool if it was reflected in your frontend as well, but since auth is not until this week, I understand you didn't want anyone to be able to delete anyone else's posts